code/object/objectsnd.h

```diff
@@ -14,7 +14,7 @@
 
 #include "gamesnd/gamesnd.h"
 
-#define	OS_USED					(1<<0)
+#define	OS_IN_USE				(1<<0)		// changed from OS_USED since it shadowed another macos macro
```

Comment:
"macos" could be misread as a typo for "macros".  Could you capitalize the term or reword the comment?

Reply:
All set